src/net/sf/memoranda/ui/PreferencesDialog.java

```diff
@@ -488,8 +488,22 @@ public void actionPerformed(ActionEvent e){
 		// Build Tab2
 		rstPanelBorder = BorderFactory.createEmptyBorder(5, 5, 5, 5);
 		resourceTypePanel.setBorder(rstPanelBorder);
-		resourcePanel.add(resourceTypePanel, BorderLayout.CENTER);
```

Comment:
if we end up getting rid of the resourceTypePanel, we should probably take all related code out of this class as well too